Makefile

```diff
@@ -436,9 +436,9 @@ $(TBB_LIB): $(TBB_BASE) $(TBB_LIBDIR)
 external_eigen: $(EIGEN_BASE)
 
 $(EIGEN_BASE):
-	git clone https://github.com/cms-externals/eigen-git-mirror $@
-	cd $@ && git checkout -b cms_branch b20a61c3a0dc9a79790cd258130a99b574662272
-	cd $@ && patch -p1 < ../../eigen_hip.patch
+	git clone -b master https://gitlab.com/libeigen/eigen.git $@
+	cd $@ && git reset --hard 011e0db31d1bed8b7f73662be6d57d9f30fa457a	# from TensorFlow 2.4.1
+	#cd $@ && patch -p1 < ../../eigen_hip.patch
```

Comment:
This version appears to work with HIP as well, so the patch can be removed
```suggestion
```

Reply:
:+1: